server/api/index.js

```diff
@@ -35,6 +35,10 @@ const callShopify = () => {
 
 io.on("connection", socket => {
 	console.log("User connected");
+	callShopify().then( data => {
+		io.emit("Total", data.totalPrice );
+	   res.sendStatus(200);
```

Comment:
`res` is not defined here so this will cause an error. You can't send a response when this is not in a request.

Reply:
Otherwise, it all looks good.

Reply:
I will remove the res.